src/components/AmountTextInput.js

```diff
@@ -49,6 +49,7 @@ function AmountTextInput(props) {
             blurOnSubmit={false}
             selection={props.selection}
             onSelectionChange={props.onSelectionChange}
+            accessibilityRole={CONST.ACCESSIBILITY_ROLE.TEXT}
```

Comment:
@aimane-chnaif It is a crucial prop from the accessibility perspective. It is needed to avoid ESlint errors since all accessibility components are required to have accessibilityRole

Reply:
@Skalakid can we change it to `presentation` which means `none` if it is just to avoid ESlint errors?

Reply:
@Skalakid accessibility* props are deprecated we are trying to change it to the `role` prop check it here: https://reactnative.dev/docs/accessibility#role